gn3/db_utils.py

```diff
@@ -35,6 +38,7 @@ def database_connection(sql_uri) -> Iterator[Connection]:
     try:
         yield connection
     except mdb.Error as _mbde:
+        logger.error("DB error encountered", exc_info=1)
```

Comment:
**TIL**: I missed `exc_info` as a usable argument. Nice! I have been using `traceback.format_exc` and `traceback.format_exception` when I wanted the exception's stack trace information. I'll try this one out too.

Thanks.